src/explore-education-statistics-common/src/modules/find-statistics/components/charts/MapBlock.tsx

```diff
@@ -1,88 +1,680 @@
+                      timeIdentifier,
+                    ),
+                  };
+                },
+                {},
+              ),
+            )
+              .sort()
+              .reverse(),
+          };
+        }
+
+        return {
+          id: `groupOption_${index}`,
+          title: '',
+          selected: '',
+          type: null,
+          options: Array<SelectOption>(),
+        };
+      },
+    );
+    */
 
     return (
-      <div>
-        {geometry && (
+      <div className="govuk-grid-row">
+        <div
+          className={classNames('govuk-grid-column-one-third')}
+          aria-live="assertive"
+        >
+          <form>
+            <div className="govuk-form-group govuk-!-margin-bottom-6">
+              <FormSelect
+                name="selectedIndicator"
+                id="selectedIndicator"
+                label="Select data to view"
+                value={selected.indicator}
+                onChange={e => this.onSelectIndicator(e.currentTarget.value)}
+                options={indicators.map(
+                  indicator => meta.indicators[indicator],
+                )}
+              />
+            </div>
+
+            <div className="govuk-form-group govuk-!-margin-bottom-6">
+              <FormSelect
+                name="selectedLocation"
+                id="selectedLocation"
+                label="Select a location"
+                value={selected.location}
+                onChange={e => this.selectLocationOption(e.currentTarget.value)}
+                options={options.location}
+              />
+            </div>
+
+            {groupOptions.map(grouping => (
+              <div
+                key={grouping.id}
+                className="govuk-form-group govuk-!-margin-bottom-6"
+              >
+                <FormSelect
+                  name="grouping"
+                  id={grouping.id}
+                  label={grouping.title}
+                  value={grouping.selected}
+                  onChange={e =>
+                    this.updateSelectedGroupOption(
+                      grouping,
+                      e.currentTarget.value,
+                    )
+                  }
+                  options={grouping.options}
+                />
+              </div>
+            ))}
+          </form>
+
+          {selected.results.length > 0 ? (
+            <div>
+              {selected.results.map(result => (
+                <div
+                  key={result.id}
+                  className="dfe-dash-tiles__tile govuk-!-margin-bottom-6"
+                >
+                  <h3 className="govuk-heading-m dfe-dash-tiles__heading">
+                    {meta.indicators[result.id].label}
+                  </h3>
+                  <p
+                    className="govuk-heading-xl govuk-!-margin-bottom-2"
+                    aria-label={meta.indicators[result.id].label}
+                  >
+                    <span>
+                      {' '}
```

Comment:
We could tighten up by making it into a single templated string. As it's split into multiple lines and interpolations, we generate excessive JSX nodes currently.